config/default.yaml

```diff
@@ -5,6 +5,7 @@ proxy:
   document_management: https://api-gateway.dev.dm.reform.hmcts.net
   print_service: https://return-case-doc.dev.ccd.reform.hmcts.net
   case_activity: http://localhost:3460
+  payments: http://localhost:8080
```

Comment:
Is this a sensible default?

Reply:
Definitely not, I'm changing it to `http://localhost:3453/payments` which is the gateway

Reply:
For the time being, I'm afraid I don't have a better one to suggest. The defaults we have are mostly based on `ccd-docker`. However, we don't have support for Payments API in Docker and we're not planning to add it at this time; so for the time being this default will have to be ad-hoc.